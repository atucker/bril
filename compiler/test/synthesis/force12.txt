# ARGS: force_sketch
x * 12
