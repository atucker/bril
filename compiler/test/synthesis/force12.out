(((x + x) + x) << h79)
(((x + x) + x) << 2)
